simpledisplay.d

```diff
@@ -1269,9 +1269,10 @@ class SimpleWindow : CapableOfHandlingNativeEvent, CapableOfBeingDrawnUpon {
 	this(NativeWindowHandle nativeWindow) {
 		version(Windows)
 			impl.hwnd = nativeWindow;
-		else version(X11)
+		else version(X11) {
 			impl.window = nativeWindow;
-		else version(OSXCocoa)
+			display = XDisplayConnection.get();
```

Comment:
Why is this there?

Reply:
actualTitle breaks otherwise if you don't have any SimpleDisplay windows and you try to get the title of an unknown window

Reply:
it should prolly just be a local variable since display can change in the event loop... maybe my mistake tho.